data/10kswap.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "10KSwap",
+  "description": "10KSwap(10kswap.com), being built on StarkNet, is an AMM protocol that advances with Ethereum. 10K aims to revolutionize the performance of the AMM protocol by leveraging the rollup feature, bringing lower fees, less friction, and ultimately better liquidity to the L2 world and advance DeFi adoption.",
```

Comment:
```suggestion
  "description": "10KSwap, being built on StarkNet, is an AMM protocol that advances with Ethereum. 10K aims to revolutionize the performance of the AMM protocol by leveraging the rollup feature, bringing lower fees, less friction, and ultimately better liquidity to the L2 world and advance DeFi adoption.",
```

Just a small suggestion to make it cleaner. 10kswap.com appears right above this text in the UI so no need to reiterate 
<img width="726" alt="Screenshot 2022-11-23 at 20 35 07" src="https://user-images.githubusercontent.com/10310633/203632283-35ea30fb-195b-4bf3-a631-cf46c80b869c.png">